### ISSUE TITLE
Synchronizing issue labels with bug report form choices

### ISSUE THREAD

**Issue by Elliria** [josephj11/Class_examples#1]
### Has this issue already been reported?

- [X] I have searched through the existing issues.

### What type of issue is this?

Enhancement

### Can this be reproduced?

Always

### Which Linux distribution did you use?

Any.

### Which AutoKey version did you use?

Any.

### Which GUI did you use?

Both

### How did you install AutoKey?

pip3

### Can you describe the issue?

The existing bug report form contains these classification choices:
* Bug
* Crash/Hang/Data loss
* Enhancement
* Feature (new)
* Performance
* UI/Usability

The AutoKey project uses these labels:
* bug
* documentation
* duplicate
* enhancement
* good first issue
* help wanted
* invalid
* question
* wontfix

What can we do to make those work together? I realize that some of the labels (like duplicate, good first issue, help wanted, invalid, and wontfix) wouldn't ever be put into the form since those are for internal use, but I'm trying to figure out if we can synchronize the rest so they're identical lists other than those internal labels.

### What are the steps to reproduce the issue?

1. File a bug report.
2. Look at the Classification choices.
3. Compare those with the repository's labels.

### Expected result:

They should perhaps be similar other than internal labels.

### Actual result:

They're mostly different.

**Comment by josephj11** [comment]
The selection of categories requires input from the devs and they won't see anything here unless we send them here.

The part above that already appears to work looks like a large improvement from the current systiem.

**Comment by Elliria** [comment]
It was your idea and it's a perfect solution to  what has been a rather awkward experience until now. I've actually improved it further in my repository, so go take a look at that one to see what you think. Also, feel free to file some fake bug reports there since it's just a temporary fork for the sole purpose of creating this form.

**Comment by Elliria** [comment]
Yes, you can make any section manadory by changing ```required: false``` to ```required: true``` in the **validations** part of any section of the form. As to whether we want to, that's up to you and the developers. I only made the checkbox at the top a requirement in my example and I thought about requiring some more of it, but it occurred to me that sometimes people won't fill out everything and then they may wonder why they can't submit the form. After all, lots of people don't read the little notes that tell them there are requirements.

Then again, you're right that if they leave certain information out, it can make a bug report difficult or impossible to follow up on. Also, some folks leave a bug report and then vanish, never to come back again, so you can't get them to give further input later. It's up to you. All of that can be changed at any time if it ends ub being a problem or if people complain about it.

**Comment by Elliria** [comment]
Last, but not least, I can put a copy of the most recent version of the form into your repository as well so you can mess around with it to your heart's content over here, too. Let me know if you'd like that one here. Or you can view the raw format of the one in my repository, copying that, replace yours with it, and commi the change.

**Comment by Elliria** [comment]
We're definitely going to need dev input on the issue that I put here and in my repository. Using the same labels all throughout would probably also make things easier for everyone.

**Comment by josephj11** [comment]
Thanks for the great work. For now, we can stick to your version on your repo(s).
I filed an issue using the form on your repo with some suggestions.
